tests/models/maskformer/test_modeling_maskformer_swin.py

```diff
@@ -395,3 +399,48 @@ def recursive_check(tuple_object, dict_object):
             tuple_inputs = self._prepare_for_class(inputs_dict, model_class, return_labels=True)
             dict_inputs = self._prepare_for_class(inputs_dict, model_class, return_labels=True)
             check_equivalence(model, tuple_inputs, dict_inputs, {"output_hidden_states": True})
+
+
+@require_torch
+class MaskFormerSwinBackboneTest(unittest.TestCase, BackboneTesterMixin):
+    all_model_classes = (MaskFormerSwinBackbone,) if is_torch_available() else ()
+    config_class = MaskFormerSwinConfig
+
+    def setUp(self):
+        self.model_tester = MaskFormerSwinModelTester(self)
+        self.config_tester = ConfigTester(self, config_class=MaskFormerSwinConfig, embed_dim=37)
+
+    def test_backbone_outputs(self):
```

Comment:
Would be nice to add a comment like: `#overwrite from comm since ....(whatever reason) ....`

Reply:
Added a comment :)